src/aat/resources/features/common/case/befta-jurisdiction2-case-creation_by_solicitor_1/Befta_Jurisdiction2_Default_Full_Case_Creation_Data_solicitor_1.td.json

```diff
@@ -0,0 +1,159 @@
+{
+  "_guid_": "Befta_Jurisdiction2_Default_Full_Case_Creation_Data_solicitor_1",
+  "_extends_": "Befta_Jurisdiction2_Case_Creation_Base_Data_Solicitor_1",
+  "users": {
+    "invokingUser": {
+      "username": "befta.caseworker.2.solicitor.1@gmail.com",
+      "password": "Pa55word11"
+    }
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers"
+    },
+    "pathVariables": {
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    },
+    "body": {
+      "data": {
+        "_extends_": "Befta_Jurisdiction2_Default_Full_Case_Data_Solicitor_1"
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
```

Comment:
Please extend it from 201 response file.

Reply:
Done.